tony-core/src/main/java/com/linkedin/tony/FrameworkRuntimeProvider.java

```diff
@@ -15,50 +15,54 @@
  */
 package com.linkedin.tony;
 
-import com.linkedin.tony.runtime.HorovodRuntime;
-import com.linkedin.tony.runtime.MXNetRuntime;
-import com.linkedin.tony.runtime.PyTorchRuntime;
-import com.linkedin.tony.runtime.StandaloneRuntime;
-import com.linkedin.tony.runtime.TFRuntime;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.ServiceLoader;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 
 public class FrameworkRuntimeProvider {
```

Comment:
@zuston it'd be good if you can write a dev guide (a readme file) if others want to support another framework in TonY.

Reply:
Good idea

Reply:
Developing doc will be attached later on another PR.